### DISCUSSION TITLE
Lieblings markdown feature

### DISCUSSION THREAD

**Discussion by wulfland** [wulfland/BootCampBasta23#4] [Polls]
Was ist euer Lieblingsfeature in markdown?

**Comment by Gorphi** [comment]
Ich seh mich nächste Woche schon alle unsere Diagramme auf mermaid syntax zu portieren 😈

**Reply by mspecht89** [reply]
Und Mermaid unterstützt sogar Emojis! ❤

**Comment by mschmicking** [comment]
codeblock is the best 😏
